docs/topics/http/urls.txt

```diff
@@ -352,60 +350,24 @@ An included URLconf receives any captured parameters from parent URLconfs, so
 the following example is valid::
 
     # In settings/urls/main.py
-    from django.conf.urls import include, url
+    from django.urls import include, path
 
     urlpatterns = [
-        url(r'^(?P<username>\w+)/blog/', include('foo.urls.blog')),
+        path('<username>/blog/', include('foo.urls.blog')),
     ]
 
     # In foo/urls/blog.py
-    from django.conf.urls import url
+    from django.urls import path
     from . import views
 
     urlpatterns = [
-        url(r'^$', views.blog.index),
-        url(r'^archive/$', views.blog.archive),
+        path('', views.blog.index),
+        path('archive/', views.blog.archive),
     ]
 
 In the above example, the captured ``"username"`` variable is passed to the
 included URLconf, as expected.
 
-Nested arguments
```

Comment:
Maybe this should be moved in the part "Advanced URL with complex patterns" I talk in my previous comment (even though I doubt a lot of people is using this...)